app/forms/participants/participant_validation_form.rb

```diff
@@ -106,8 +96,22 @@ def add_mentor_info_choice
       errors.add(:do_you_want_to_add_mentor_information_choice, :blank) unless add_mentor_information_choices.map(&:id).include?(do_you_want_to_add_mentor_information_choice)
     end
 
-    def trn_choice
-      errors.add(:do_you_know_your_trn_choice, :blank) unless trn_choices.map(&:id).include?(do_you_know_your_trn_choice)
+    def trn_entry
+      @trn = trn&.squish
+
+      if trn.blank?
+        errors.add(:trn, :blank)
+      else
+        @trn = trn.gsub(/RP/, "").gsub(/[\/\s]/, "")
```

Comment:
What's this about?

Reply:
TRN can contain "RP" and "/" we were asked if we could accept that and strip them out to make it easier for the user to enter the right thing.